feat(JS CDN): Add async/defer note (#1404)
getsentry/sentry-docs: src/collections/_documentation/error-reporting/getting-started-install/browser.md

```

{% wizard hide %}
{% capture __alert_content -%}

> It's possible to include `defer` in your script tag, but keep in mind that any errors which occur in scripts which execute before the browser SDK script executes won’t be caught (because the SDK won’t be initialized yet). It is therefore recommended that if you use `defer`, you a) place the script tag for the browser SDK first, and b) mark it, and all of your other scripts, `defer` (but not `async`), thereby guaranteeing that it’s executed before any of the others.

Reviewer: ```suggestion
It's possible to include `defer` in your script tag, but keep in mind that any errors which occur in scripts that execute before the browser SDK script executes won’t be caught (because the SDK won’t be initialized yet). We strongly recommend that if you use `defer`, you a) place the script tag for the browser SDK first, and b) mark it, and all of your other scripts, `defer` (but not `async`), thereby guaranteeing that it’s executed before any of the others.
```
